ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java

```diff
@@ -57,47 +57,46 @@
  * Ignite abstract client.
  */
 public abstract class IgniteAbstractClient extends DB {
+  protected static final String HOSTS_PROPERTY = "hosts";
 
-  private static final Logger LOG = LogManager.getLogger(IgniteAbstractClient.class);
+  protected static final String PRIMARY_COLUMN_NAME = "ycsb_key";
 
-  protected static String cacheName;
+  protected static final String DEFAULT_ZONE_NAME = "Z1";
 
-  protected static int fieldCount;
+  protected static final String DEFAULT_STORAGE_PROFILE_NAME = "default";
 
-  protected static String fieldPrefix;
+  protected static final String DEFAULT_COLUMNAR_PROFILE_NAME = "myColumnarStore";
 
-  protected static long recordsCount;
+  protected static final long TABLE_CREATION_TIMEOUT_SECONDS = 10L;
 
-  protected static long batchSize;
+  private static final Logger LOG = LogManager.getLogger(IgniteAbstractClient.class);
 
-  protected static final List<String> FIELDS = new ArrayList<>();
+  protected String cacheName;
 
-  protected static final String HOSTS_PROPERTY = "hosts";
+  protected int fieldCount;
 
-  protected static final String PRIMARY_COLUMN_NAME = "ycsb_key";
-
-  protected static final String DEFAULT_ZONE_NAME = "Z1";
+  protected String fieldPrefix;
 
-  protected static final String DEFAULT_STORAGE_PROFILE_NAME = "default";
+  protected long recordsCount;
```

Comment:
I do not think that it is a good idea to remove `static` access modifier from declaration of fields. It is used as a simple way to share the same field across all benchmarking threads. Most YCSB benchmarks for other DBs follow the same pattern. So let's keep that.

Reply:
Also, the `recordsCount` variable is initialized only once in the `initProperties` method. So only 1 thread among N will have this field initialized.

Reply:
But we need it as instance variables to make unit test. I've move initProperties before the init lock to initialize that properties on each instance creation.